playbooks/common/openshift-glusterfs/config.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Open firewall ports for GlusterFS
+  hosts: oo_glusterfs_to_config
+  vars:
+    os_firewall_allow:
+    - service: glusterfs_sshd
+      port: "2222/tcp"
+    - service: glusterfs_daemon
+      port: "24007/tcp"
+    - service: glusterfs_management
+      port: "24008/tcp"
+    - service: glusterfs_bricks
+      port: "49152-49251/tcp"
+  roles:
+  - os_firewall
+
+- name: Configure GlusterFS
+  hosts: oo_first_master
+  roles:
+  - role: openshift_storage_glusterfs
```

Comment:
@abutcher Good call. Do we need the default given [this](https://github.com/jarrpa/openshift-ansible/blob/86ec8c92ba5f99a2914f863d7a56a7aab406a64d/playbooks/common/openshift-cluster/evaluate_groups.yml#L127-L134)?